### ISSUE TITLE
Connect string should be in bacalhau.run

### ISSUE THREAD

**Issue by aronchick** [bacalhau-project/bacalhau#3323]
If your logs go by too fast, you won't get the string you're supposed to run.

**Comment by frrist** [comment]
It should be in the repo:
1 `bacalhau serve`
2. `cat ~/.bacalhau/bacalhau.run`

**Comment by aronchick** [comment]
it's not.

**Comment by aronchick** [comment]
here's what's in `bacalhau run`:

```
export BACALHAU_NODE_IPFS_SWARMADDRESSES=/ip4/127.0.0.1/tcp/62973/p2p/QmT2My4vP4ALjMsQM6ppWxxndjD7eGaDYAtZF1vDSVsygG
export BACALHAU_NODE_CLIENTAPI_HOST=0.0.0.0
export BACALHAU_NODE_CLIENTAPI_PORT=1234
export BACALHAU_NODE_LIBP2P_PEERCONNECT=/ip4/100.100.225.91/tcp/1235/p2p/Qmcb9tHT7MbM89Lqdb9QVtVWcinCbPxKiBQGrqirsREwXu
```

Here's the connect string:
```
bacalhau serve --node-type=compute --private-internal-ipfs --peer=/ip4/100.100.225.91/tcp/1235/p2p/Qmcb9tHT7MbM89Lqdb9QVtVWcinCbPxKiBQGrqirsREwXu --ipfs-swarm-addrs=/ip4/127.0.0.1/tcp/60511/p2p/QmZ89xCcCHrA6Am5TjzCdvkgfXfDaSb2VUp4CUzkPcPKJu
```

**Comment by wdbaruni** [comment]
similar to https://github.com/bacalhau-project/bacalhau/issues/2522, we should deprecate bacalhau.run and relying on a connect string

**Comment by wdbaruni** [comment]
to be addressed as part of #4014
